projects/external-components/src/lib/primeng-selectors/components/generate-with-ai/generate-with-ai.component.html

```diff
@@ -5,12 +5,21 @@
         <div *ngIf="part.type === 'text'" class="message-text">{{ part.content }}</div>
         <div *ngIf="part.type === 'code'"  class="code-holder ">
           <div class="button-code">
-            <div class="language-code">typescript</div>
-            <div class="code-import-button" (click)="copycode( part.content)">
-              <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24" class="icon-sm">
-                <path fill="currentColor" fill-rule="evenodd" d="M7 5a3 3 0 0 1 3-3h9a3 3 0 0 1 3 3v9a3 3 0 0 1-3 3h-2v2a3 3 0 0 1-3 3H5a3 3 0 0 1-3-3v-9a3 3 0 0 1 3-3h2zm2 2h5a3 3 0 0 1 3 3v5h2a1 1 0 0 0 1-1V5a1 1 0 0 0-1-1h-9a1 1 0 0 0-1 1zM5 9a1 1 0 0 0-1 1v9a1 1 0 0 0 1 1h9a1 1 0 0 0 1-1v-9a1 1 0 0 0-1-1z" clip-rule="evenodd"></path>
-              </svg>
-               Copy Code</div>
+            <div class="language-code">Typescript</div>
+            <div class="button-actions">
+              <div class="code-import-button" (click)="copycode(part.content)" pTooltip="Copy Code to Clipboard" tooltipPosition="top">
```

Comment:
Add a functionality to change label of button from copy code to copied for few seconds.

Reply:
done